spec/unit/actions/synchronize_cas_spec.rb

```diff
@@ -0,0 +1,59 @@
+# encoding: UTF-8
+#
+# Copyright (c) 2010-2017 GoodData Corporation. All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+describe GoodData::LCM2::SynchronizeComputedAttributes do
+  let(:gdc_gd_client) { double(GoodData::Rest::Client) }
+  let(:logger) { double(Logger) }
+  let(:project) { double(GoodData::Project) }
+  let(:synchronize) do
+    [{ from: 'foo', to: [{ ca_scripts: { 'maqlDdlChunks' => ['expected_maql'] } }] }]
+  end
+  let(:basic_params) do
+    {
+        gdc_gd_client: gdc_gd_client,
+        development_client: gdc_gd_client,
+        synchronize: synchronize,
+        gdc_logger: logger
+    }
+  end
+  let(:params) { basic_params }
+  let(:converted_params) do
+    GoodData::LCM2.convert_to_smart_hash(params)
+  end
+
+  before do
```

Comment:
Identical code found in synchronize_cas_spec.rb:27, synchronize_ldm_spec.rb:27 (mass = 100)